### ISSUE TITLE
unmatched pathItems in service-based-mockups

### ISSUE THREAD

**Issue by commonism** [SNIA/Swordfish-API-Emulator#137]
Validating http://service-based-mockups.swordfishmockups.com fails due to missing PathItems for 
```
/redfish/v1/StorageServices/ISC/StoragePools/SASPoolR5/AllocatedVolumes/3/CapacitySources/Source1/ProvidingPools
/redfish/v1/StorageServices/ISC/StoragePools/SASPoolR5/AllocatedVolumes/3/CapacitySources/Source1/ProvidingPools
/redfish/v1/StorageServices/ISC/StoragePools/SASPool/AllocatedVolumes/1/CapacitySources/Source1/ProvidingPools
/redfish/v1/StorageServices/ISC/StoragePools/SASPool/AllocatedVolumes/1/CapacitySources/Source1/ProvidingPools
/redfish/v1/StorageServices/ISC/StoragePools/SATAPool/AllocatedVolumes/2/CapacitySources/Source1/ProvidingPools
/redfish/v1/StorageServices/ISC/StoragePools/SATAPool/AllocatedVolumes/2/CapacitySources/Source1/ProvidingPools
/redfish/v1/StorageServices/2/StoragePools/BasePool/AllocatedVolumes/3/CapacitySources/Source1/ProvidingVolumes
/redfish/v1/StorageServices/2/StoragePools/BasePool/AllocatedVolumes/3/CapacitySources/Source1/ProvidingVolumes
```
in the swordfish v.1.2.6 openapi.yaml.

**Comment by rahlvers** [comment]
Cross-logged in Swordfish TWG repository.

**Comment by rahlvers** [comment]
There are two redfish.uri annotations missing in the underlying schema that the openapi.yaml file is generated from.  This fix is slated to be included in Swordfish 1.2.8 release.
